kube.libsonnet

```diff
@@ -577,6 +577,16 @@
     assert std.length(rel_paths) == 0 : "paths must be absolute: " + rel_paths,
   },
 
```

Comment:
Naming wise, I'd prefer these cloud specific additions to be name-scoped, please add them as:
```
  gke:: {
    ManagedCertificates(name)  { ... },
    BackendConfig(name) { ... },
  },
```
Using them would then be:
```
  my_cert: kube.gke.ManagedCertificate("my-cert") {
    spec+: {
       domains: ["foo.example.com"],
    },
  }
```